docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -17,55 +17,155 @@ while a bigger model can be deployed on Azure for higher precision. Using the Az
 
 Azure Execution Provider is in preview stage, and all API(s) and usage are subject to change.
 
+Till onnxruntime 1.16, three operators are available:
```

Comment:
'Till 1.16'  what about what you can do now?   Seems an odd opening

Reply:
How about the following? 

Starting 1.16 release AzureEP supports 3 new models hosted on both Azure and OpenAI using pluggable custom ops. Each custom op targets a specific model and can be used to compose model configurations that support various scenarios. As an example, we have written samples to show configurations that support If/Then/And scenarios. More info about these can be found here: https://github.com/microsoft/onnxruntime-inference-examples/tree/main/python/AzureEP.

Read more about these custom ops and the models they support here:
- [OpenAIAudioToText](https://github.com/microsoft/onnxruntime-extensions/blob/main/docs/custom_ops.md#openaiaudiototext)
- [AzureTextToText](https://github.com/microsoft/onnxruntime-extensions/blob/main/docs/custom_ops.md#azuretexttotext)
- [AzureTritonInvoker](https://github.com/microsoft/onnxruntime-extensions/blob/main/docs/custom_ops.md#azuretritoninvoker)

### Asynchronous Run API
Moreover, we've also added an [asynchronous Run API](https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_c_api.h#L4341) to make inferences asynchronously. This will useful when inferencing models that are hosted on remote services like Azure and OpenAI.

Reply:
"As an example, we have written samples to show configurations that support If/Then/And scenarios. More info about these can be found here: https://github.com/microsoft/onnxruntime-inference-examples/tree/main/python/AzureEP." 
Did this part in Usage under the 2nd mode.

"Moreover, we've also added an [asynchronous Run API](https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_c_api.h#L4341) to make inferences asynchronously. This will useful when inferencing models that are hosted on remote services like Azure and OpenAI."

Did this in Usage, under the 1st mode.